model/firewall.rb

```diff
@@ -17,7 +17,29 @@ def insert_firewall_rule(cidr, port_range)
       port_range: port_range
     )
 
-    private_subnet&.incr_update_firewall_rules
+    private_subnets.map(&:incr_update_firewall_rules)
```

Comment:
nit: no need to returned object
```suggestion
    private_subnets.each(&:incr_update_firewall_rules)
```